var/spack/repos/builtin/packages/magics/package.py

```diff
@@ -84,50 +122,51 @@ def patch(self):
             filter_file('#!/usr/bin/python', '#!/usr/bin/env python', pyfile)
 
     def cmake_args(self):
-        spec = self.spec
         args = [
             '-DENABLE_ODB=OFF',
-            '-DENABLE_PYTHON=OFF',
-            '-DBOOST_ROOT=%s' % spec['boost'].prefix,
```

Comment:
What happened to `BOOST_PATH`?

Reply:
It's unnecessary since Spack sets CMAKE_PREFIX_PATH and the directories that are specified by the variable are the first to be checked by cmake.

Reply:
Got it!